pkg/cmd/events/event_filter.go

```diff
@@ -91,6 +98,49 @@ func (f *FilterByReasons) FilterEvents(events ...*corev1.Event) []*corev1.Event
 	return ret
 }
 
+type FilterByAround struct {
+	Around         string
+	AroundDuration time.Duration
+}
+
+func (f *FilterByAround) FilterEvents(events ...*corev1.Event) []*corev1.Event {
+	t := events[len(events)-1].LastTimestamp.Time
+	aroundParts := strings.Split(f.Around, ":")
```

Comment:
isn't there a clever parser for times in Go?

Reply:
i found nothing in standard go library